### PR TITLE
update type to number

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe, LGTY?

### CODE REVIEW

File: spec/inbound_spec.coffee [hunk 1]
```diff
@@ -129,12 +129,8 @@ describe 'Inbound', ->
...
 
     describe 'Loan', ->
-      it 'should parse amount', ->
-
-      it 'should parse amount when range', ->
-        assert.equal integration.request(req(body(loan_amount: '1000-2000'))).mortgage.loan.amount, '1000-2000'
 
-      it 'should parse amount when number', ->
+      it 'should parse amount', ->
         assert.equal integration.request(req(body(loan_amount:28000))).mortgage.loan.amount, 28000
```
Comment (alexkwolfe) [inline spec/inbound_spec.coffee]: I'm confused why the integration is providing an integer here. I don't see any use of parseInt in for this field...

**Comment by ros3bud** [comment]
Thanks for catching that. I updated the test, it should be a string because that is how the data will be coming in.

**Comment by ros3bud** [comment]
@alexkwolfe, ^^

**Comment by alexkwolfe** [comment]
LGTM

**Comment by ros3bud** [comment]
Updated master with these changes via cherry pick .

**Pull request closed without merging**